packages/tools/fluid-runner/package.json

```diff
@@ -61,9 +95,11 @@
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --ignore \"dist/test/types/*\" --recursive dist/test -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha": "npm run test:mocha:cjs && echo \"ESM skipped: see AB#7518 - npm run test:mocha:cjs\"",
```

Comment:
If we decide to keep ESM, we should finish converting the tests, which we will likely need help from the owners to do (AB#7518).